homework/index.js

```diff
@@ -29,22 +29,51 @@
     return elem;
   }
 
+  function changeDateTimeFormat(dateTime) {
+    const timeFormat = new Date(dateTime);
+    return timeFormat.toLocaleString();
+  }
+  function makeTable(table, header, value) {
```

Comment:
You are not making a table here. You are creating or adding a table row. This should be reflected in the function name.

(Also, place a blank line before a function name to make it stand out in the code. This is like placing blanks lines between paragraphs in a piece of text.)